crates/bevy_render/src/camera/camera.rs

```diff
@@ -112,3 +112,24 @@ pub fn camera_system<T: CameraProjection + Component>(
         }
     }
 }
+
+#[derive(Reflect)]
```

Comment:
Most of these structs (if possible) should also derive `Debug`

Reply:
Ok, I will do.